docs/_templates/layout.html

```diff
@@ -38,154 +38,154 @@
                     <dl>
                         <dt>Versions</dt>
 
-                        <dd><a href="https://docs.wagtail.io/en/latest/">latest</a></dd>
+                        <dd><a href="https://docs.wagtail.org/en/latest/">latest</a></dd>
```

Comment:
Since this is a "testing only" template (per comments above), would it be better to replace these with links to example.com? This list of links here seems suspiciously specific/hard-coded.